compiler/src/dmd/cli.d

```diff
@@ -345,6 +345,10 @@ dmd -cov -unittest myprog.d
                 $(LI $(I all): All)
             )`,
         ),
+        Option("-edition[=<NNNN>G[<filename>]]",
+            "set language edition to edition year, apply to $(I filename)",
```

Comment:
yes the failure is legit, search the output for this line and you will see it verbatim, without the `$(I filename)` converted into HTML Italics.

Also, remove the leading `-` in `-edition` on the line above.